Creation of a learning pathway for the creation of a sig and codex (#6128)
galaxyproject/training-material: topics/community/tutorials/community-tutorial-table/tutorial.md

The generated community-specific table can be used as it and/or embedded, e.g. into the respective Galaxy Hub page or Galaxy subdomain.

The pipeline is **fully automated** and executes on a **weekly** basis. Any research community can apply the pipeline to create a table specific to their community.

The aim is this tutorial is to create a tutorial table for a community.

> <agenda-title></agenda-title>
>
> In this tutorial, we will cover:
>
> 1. TOC
> {:toc}
>
{: .agenda}

# Pull list of tags relevant to your community

To add tutorials in your community tutorial table, you will need to indicate a list of tags relevant to your community, and tutorials associated with this tag will be automatically pulled from the [Galaxy Training Network](https://training.galaxyproject.org/). Only tutorials with the selected tags will be added to the filtered table.
You will then be able to remove tutorials that are not relevant to your community or deprecated.

> <hands-on-title>Select tutorials tags from the Galaxy Training Network</hands-on-title>
>

> > 1. Go to the [Galaxy Training Network](https://training.galaxyproject.org/)

Reviewer: ```suggestion
> 1. Go to the [Galaxy Training Network]({% link index.md %})
```

> > 2. Browse for tutorials that are releavnt to your community

Reviewer: ```suggestion
> 2. Browse for tutorials that are relevant to your community
```

> 3. Note the tags that are associated with tutorials of interest
{: .hands_on}

> Note : You can also create a tag (for example, your community name) and tag all relevant tutorials with it. Then, you only need to use this tag to pull all the relevant tutorials.

Reviewer: Could put that as a comment box? Thanks

---
layout: tutorial_hands_on
title: Creation of a Galaxy tutorial table for your community
level: Introductory
redirect_from:
- /topics/dev/tutorials/community-tutorial-table/tutorial
subtopic: sig

questions:
- Is it possible to have an overview of all Galaxy tutorials for a specific scientific domain?
objectives:
- Create a community reviewed table for Galaxy tutorials within a specific scientific domain
- Embed the tutorial table in a community page
time_estimation: 1H
key_points:
- The Galaxy Codex extracts all Galaxy tutorials to create tables
- The community Galaxy tutorials table can be embed into any website
tags:
- Community
- SIG
contributions:
  authorship:
    - bebatut
    - paulzierep
    - scorreard

---

Similarly to the numerous tools available on Galaxy, the [Galaxy Training Network](https://training.galaxyproject.org/) includes numerous tutorials. This tutorail will take you through the steps to generate resources listing all the relevant tutorials and display the tutorial table on your community codex page.

> The pipeline creates table with all the tutorials. This table can be **filtered to only include tutorials that are relevant to a specific research community**.

Reviewer 2: ```suggestion
The pipeline creates a table with all the tutorials. This table can be **filtered to only include tutorials that are relevant to a specific research community**.
```

> The generated community-specific table can be used as it and/or embedded, e.g. into the respective Galaxy Hub page or Galaxy subdomain.

Reviewer 2: As in `topics/community/tutorials/community-tool-table/tutorial.md` above, I am unsure what 'it' in "community-specific table can be used as *it* " refers to.

> 8. Click on `Create a new branch for this commit and start a pull request`
> 9. Create the pull request by following the instructions
>
{: .hands_on}

# Pull list of tags relevant to your community

To add tutorials in your community tutorial table, you will need to indicate a list of tags relevant to your community, and tutorials associated with this tag will be automatically pulled from the [Galaxy Training Network](https://training.galaxyproject.org/) one a week. Only tutorials with the selected tags will be added to the filtered table.
You will then be able to remove tutorials that are not relevant to your community or deprecated.

> <hands-on-title>Select tutorials tags from the Galaxy Training Network</hands-on-title>
>
> 1. Go to the [Galaxy Training Network]({% link index.md %})
> 2. Browse for tutorials that are relevant to your community
> 3. Note the tags that are associated with tutorials of interest
{: .hands_on}
> <comment-title>Tips</comment-title>
> You can also create a tag (for example, your community name) and tag all relevant tutorials with it. Then, you only need to use this tag to pull all the relevant tutorials.
{: .comment}

> # Add the list releavant tags for your community in the tutorial_tags file

Reviewer 2: ```suggestion
# Add the list relevant tags for your community in the `tutorial_tags` file
```

> <hands-on-title>Add the relevant tags to the tutorial_tags file</hands-on-title>
> 1. Open or create a file named `tutorial_tags` in your comunity metadata folder (`communities/<your community>/metadata/tutorial_tags`)
> 2. Add the name of the tags relevant to your community in the `tutorial_tags` file, with 1 tag per row
>
>    For example:
>    ```
>    Assembly
>    Metagenomics
>    ```

> {: .hands_on}

Reviewer 2: As above in `topics/community/tutorials/community-tool-table/tutorial.md`, it is useful to know *what* tags these are. ToolShed tags?

The format example is good, but the user is unsure where to find the valid tags.

---
layout: tutorial_hands_on
title: Creation of a Galaxy tutorial table for your community
level: Introductory
redirect_from:
- /topics/dev/tutorials/community-tutorial-table/tutorial
subtopic: sig

questions:
- Is it possible to have an overview of all Galaxy tutorials for a specific scientific domain?
objectives:
- Create a community reviewed table for Galaxy tutorials within a specific scientific domain
- Embed the tutorial table in a community page
time_estimation: 1H
key_points:
- The Galaxy Codex extracts all Galaxy tutorials to create tables
- The community Galaxy tutorials table can be embedded into any website
tags:
- Community
- SIG
contributions:
  authorship:
    - bebatut
    - paulzierep
    - scorreard

>     - BirdmanRidesAgain
>     - BirdmanRidesAgain

Reviewer 3: oh, it was duplicated

```suggestion
    - BirdmanRidesAgain
```
